### PR TITLE
[Merged by Bors] - fixing call filter governance and setup

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
Signed-off-by: Dzmitry Lahoda <dzmitry@lahoda.pro>

## Description
* call filter should not filter out democracy
* and it should have proper origin setup
* and picasso should have it in runtime

### CODE REVIEW

File: runtime/composable/src/lib.rs [hunk 1]
```diff
@@ -749,6 +749,7 @@ pub struct BaseCallFilter;
 
 impl Contains<Call> for BaseCallFilter {
 	fn contains(call: &Call) -> bool {
+		// TODO: not up to date as whole composable setup
```
Comment (KaiserKarel) [inline runtime/composable/src/lib.rs:752]: What do you mean by this

File: runtime/dali/src/lib.rs [hunk 2]
```diff
@@ -798,24 +800,16 @@ impl pallet_staking_rewards::Config for Runtime {
...
 pub struct BaseCallFilter;
-
 impl Contains<Call> for BaseCallFilter {
 	fn contains(call: &Call) -> bool {
-		if call_filter::Pallet::<Runtime>::contains(call) {
-			return false
-		}
-		!matches!(call, Call::Tokens(_) | Call::Indices(_) | Call::Democracy(_) | Call::Treasury(_))
+		!(call_filter::Pallet::<Runtime>::contains(call) ||
+			matches!(call, Call::Tokens(_) | Call::Indices(_) | Call::Treasury(_)))
```
Comment (KaiserKarel) [inline runtime/dali/src/lib.rs:806]: Should we add `Assets` to this and filter tokens?

**Review by KaiserKarel — APPROVED** [review]

File: runtime/dali/src/lib.rs (hunk 2 above)
Comment (dzmitry-lahoda) [reply runtime/dali/src/lib.rs:806]: i removed only democracy, other than that filter is same. it is clear why tokens are filtered out. assets should be not as i think. balances? may or may not be, anyway they sudo secured.

File: runtime/composable/src/lib.rs (hunk 1 above)
Comment (dzmitry-lahoda) [reply runtime/composable/src/lib.rs:752]: i do not updtae composable runtime. and not updatwd democracy and xcmp here too. and others do not update it too. so likely need at some point to recheck whole runtime and update.

**Review by alexkdefi — APPROVED** [review]
👍

**Review by andor0 — APPROVED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Pull request closed without merging**